src/ophys_etl/workflows/ophys_experiment.py

```diff
@@ -45,7 +45,8 @@ def get_ophys_experiment_ids(
 
     def has_completed_workflow_step(
         self,
-        workflow_step: WorkflowStepEnum
+        workflow_step: WorkflowStepEnum,
+        passed_or_qc_only: bool = True
```

Comment:
Should we default this to false as the I think the only process that would want this to be True is nway_matching.

Reply:
I'm using the same default as defined in `get_ophys_experiment_ids`